### PR TITLE
Add gauge metrics to docs

### PR DESCRIPTION
**Pull request by SugandhaAgrawal**
## Description
Since we have introduced `gauge` in Openwhisk, addition to the documentation for the metrics is done here.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [x] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [x] I updated the documentation where necessary.

### CODE REVIEW

**Comment by SugandhaAgrawal** [comment]
@chetanmeh - Can you please have a look? TIA

File: docs/metrics.md [hunk 1]
```diff
@@ -96,6 +96,10 @@ Histogram record the [distribution](http://kamon.io/documentation/0.6.x/kamon-co
 * `my_metric.max` - Max of aggregated values during the flush interval.
 * `my_metric.min` - Min of aggregated values during the flush interval.
 
+#### Gauges
+
+Gauges record the [distribution](http://kamon.io/documentation/0.6.x/kamon-core/metrics/instruments/#gauges) of given metric and there names are prefixed with `openwhisk.gauge`. For example `openwhisk.gauge.loadbalancer_totalHealthyInvoker_count`. A gauge metrics provides the value at the given point and reports the same data unless the value has been changed be incremental or decremental than before. Gauges are useful for reporting metrics like kafka queue size or disk size.
```
Comment (chetanmeh) [inline docs/metrics.md]: Better to use latest docs link https://kamon.io/docs/latest/core/metrics/

Comment (SugandhaAgrawal) [reply docs/metrics.md]: I will update

**Comment by rabbah** [comment]
@chetanmeh we can merge this now?

**Comment by asfgit** [comment]
Can one of the admins verify this patch?

**Pull request merged**